scripts/processDevPackageOnly.cmd

```diff
@@ -1,2 +0,0 @@
-powershell -NoProfile -ExecutionPolicy Unrestricted -Command "& '%~dp0processTemplates.ps1' dev"
-exit /B %ERRORLEVEL%
```

Comment:
what is the new version of this?

we use this in the testing instructions to allow people to generate a local gallery:
[https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/Contributing.md#deploying-your-own-gallery](https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/Contributing.md#deploying-your-own-gallery)

teams like WLI are using this to build an en-us only package they can deploy to storage to test/preview changes.  does the new process have a way to do this?
whatever it is you'll need to update the above doc/process as well.

Reply:
Added back processDevPackageOnly.cmd that generates en-us package. Kept the name so I don't have to change the docs :)